client/lib/titan/get-titan-urls.js

```diff
@@ -4,20 +4,21 @@
 const TITAN_APPS = {
 	CALENDAR: 'calendar',
 	CONTACTS: 'contacts',
-	EMAIL: 'email',
+	MAIL: 'mail',
```

Comment:
This url path is more inter-operable between the staging and production platforms. The previous string doesn't work with staging.

Reply:
This is nitpicking but maybe we could keep the original constant though? `mail` is rather used to denote a _postal delivery_:

```suggestion
	EMAIL: 'mail',
```

Reply:
Sure